File: c/11a.c
#include <stdio.h>
#include <stdlib.h>
int main(){
    int userAns;
    printf("What is 24+9?\n");
    scanf("%d", &userAns);
    if (userAns == 24+9){ // just incase the answer of 24+9 changes in the future 
        printf("\nwell done this is the correct answer.");
    } else {
        printf("\nwrong answer.");
    }
}

File: dart/11a.dart
//11a
import 'dart:io';

void main(){
  //11a
  String userAns = stdin.readLineSync()!;
  // use 24+9 incase it is ever not 33
  if (userAns == "33"){
    print("well done this is the correct answer");
  } else{
    print("WRONG");
  }
}